### PR TITLE
Update content about upgrade

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by lilyjazz** [comment]
tech review @lucklove

File: upgrade-tidb-using-tiup.md [hunk 1]
```diff
@@ -5,20 +5,21 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
...
-本文档适用于使用 TiUP 从 TiDB 4.0 版本升级至 TiDB 5.0 版本，以及从 5.0 版本升级至后续修订版本。
+本文档适用于使用 TiUP 从 TiDB 4.0 版本升级至 TiDB 5.0 版本及后续修订版本，以及从 5.0 版本升级至后续修订版本。
 
 如果原集群是 3.0 或 3.1 或更老的版本，需要先升级到 4.0 后再升级到 5.0，不可跨主版本升级。
 
 ## 1. 升级兼容性说明
 
-- 不支持在升级后回退至 4.0 或更旧版本。
-- 使用 TiDB Ansible 管理的 4.0 版本集群，需要先按照 [4.0 版本文档的说明](https://docs.pingcap.com/zh/tidb/v4.0/upgrade-tidb-using-tiup)将集群导入到 TiUP (`tiup cluster`) 管理后，再按本文档说明升级到 5.0 版本。
+- TiDB 目前暂时不支持版本降级或升级后回退，这类功能将在未来支持。
```
Comment (qiancai) [inline upgrade-tidb-using-tiup.md:14]:
```suggestion
- TiDB 目前暂不支持版本降级或升级后回退，此类功能将在未来支持。
```

File: upgrade-tidb-using-tiup.md [hunk 2]
```diff
@@ -5,20 +5,21 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
...
+- 使用 TiDB Ansible 管理的 4.0 版本集群，需要先按照 [4.0 版本文档的说明](https://docs.pingcap.com/zh/tidb/v4.0/upgrade-tidb-using-tiup)将集群导入到 TiUP (`tiup cluster`) 管理后，再按本文档说明升级到 5.0 版本及后续修订版本。
 - 若要将 3.0 之前的版本升级至 5.0 版本：
```
Comment (qiancai) [inline upgrade-tidb-using-tiup.md:16]: 如果此文档里有哪些可以从 5.0. 替换为 5.1 的地方，麻烦请替换下吧。因为 v5.1 文档在官网有一个独立的入口，此文档是专门给 v5.1 用的。

File: upgrade-tidb-using-tiup.md [hunk 3]
```diff
@@ -248,4 +249,4 @@ tiup install ctl:v5.0.0
 ## 5. TiDB 5.1 兼容性变化
 
 - 兼容性变化请参考 5.1 GA Release Notes。
```
Comment (qiancai) [inline upgrade-tidb-using-tiup.md:251]:
```suggestion
- 兼容性变化请参考 5.1 Release Notes。
```

File: upgrade-tidb-using-tiup.md [hunk 4]
```diff
@@ -5,20 +5,24 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
...
-- 不支持在升级后回退至 4.0 或更旧版本。
-- 使用 TiDB Ansible 管理的 4.0 版本集群，需要先按照 [4.0 版本文档的说明](https://docs.pingcap.com/zh/tidb/v4.0/upgrade-tidb-using-tiup)将集群导入到 TiUP (`tiup cluster`) 管理后，再按本文档说明升级到 5.0 版本。
+- TiDB 目前暂不支持版本降级或升级后回退，此类功能将在未来支持。
+- 使用 TiDB Ansible 管理的 4.0 版本集群，需要先按照 [4.0 版本文档的说明](https://docs.pingcap.com/zh/tidb/v4.0/upgrade-tidb-using-tiup)将集群导入到 TiUP (`tiup cluster`) 管理后，再按本文档说明升级到 5.0 版本及后续修订版本。
 - 若要将 3.0 之前的版本升级至 5.0 版本：
     1. 首先[通过 TiDB Ansible 升级到 3.0 版本](https://docs.pingcap.com/zh/tidb/v3.0/upgrade-tidb-using-ansible)。
     2. 然后按照 [4.0 版本文档的说明](https://docs.pingcap.com/zh/tidb/v4.0/upgrade-tidb-using-tiup)，使用 TiUP (`tiup cluster`) 将 TiDB Ansible 配置导入。
     3. 将集群升级至 4.0 版本。
-    4. 按本文档说明将集群升级到 5.0 版本。
+    4. 按本文档说明将集群升级到 5.0 版本及后续修订版本。
```
Comment (lilyjazz) [inline upgrade-tidb-using-tiup.md:23]:
```suggestion
    4. 按本文档说明将集群升级到 5.1 版本。
```

File: upgrade-tidb-using-tiup.md [hunk 5]
```diff
@@ -5,20 +5,24 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
...
+>  **注意：**
+>  如果原集群是 3.0 或 3.1 或更老的版本，不支持直接升级到 5.1 及后续修订版本。你需要先从老版本升级到 4.0 后，再从 4.0 升级到 5.1 及后续修订版本。
 
 ## 1. 升级兼容性说明
 
-- 不支持在升级后回退至 4.0 或更旧版本。
-- 使用 TiDB Ansible 管理的 4.0 版本集群，需要先按照 [4.0 版本文档的说明](https://docs.pingcap.com/zh/tidb/v4.0/upgrade-tidb-using-tiup)将集群导入到 TiUP (`tiup cluster`) 管理后，再按本文档说明升级到 5.0 版本。
+- TiDB 目前暂不支持版本降级或升级后回退，此类功能将在未来支持。
+- 使用 TiDB Ansible 管理的 4.0 版本集群，需要先按照 [4.0 版本文档的说明](https://docs.pingcap.com/zh/tidb/v4.0/upgrade-tidb-using-tiup)将集群导入到 TiUP (`tiup cluster`) 管理后，再按本文档说明升级到 5.0 版本及后续修订版本。
 - 若要将 3.0 之前的版本升级至 5.0 版本：
```
Comment (lilyjazz) [inline upgrade-tidb-using-tiup.md:19]:
```suggestion
- 若要将 3.0 之前的版本升级至 5.1 版本：
```

File: upgrade-tidb-using-tiup.md (hunk 2 above)
Comment (lilyjazz) [reply upgrade-tidb-using-tiup.md]: 修改完了，可以帮忙 check 下～
具体的技术检查还是需要帅总 @lucklove

File: upgrade-tidb-using-tiup.md [hunk 6]
```diff
@@ -5,20 +5,24 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
...
+- TiDB 目前暂不支持版本降级或升级后回退，此类功能将在未来支持。
+- 使用 TiDB Ansible 管理的 4.0 版本集群，需要先按照 [4.0 版本文档的说明](https://docs.pingcap.com/zh/tidb/v4.0/upgrade-tidb-using-tiup)将集群导入到 TiUP (`tiup cluster`) 管理后，再按本文档说明升级到 5.1 版本及后续修订版本。
+- 若要将 3.0 之前的版本升级至 5.1 版本：
     1. 首先[通过 TiDB Ansible 升级到 3.0 版本](https://docs.pingcap.com/zh/tidb/v3.0/upgrade-tidb-using-ansible)。
     2. 然后按照 [4.0 版本文档的说明](https://docs.pingcap.com/zh/tidb/v4.0/upgrade-tidb-using-tiup)，使用 TiUP (`tiup cluster`) 将 TiDB Ansible 配置导入。
     3. 将集群升级至 4.0 版本。
-    4. 按本文档说明将集群升级到 5.0 版本。
+    4. 按本文档说明将集群升级到 5.1 版本。
 - 支持 TiDB Binlog，TiCDC，TiFlash 等组件版本的升级。
+- 具体不同版本的兼容性说明，请查看各个版本的 [Release Note](https://docs.pingcap.com/zh/tidb/dev/release-notes)。请根据各个版本的 Release Note 的兼容性更改调整集群的配置。
```
Comment (qiancai) [inline upgrade-tidb-using-tiup.md:25]:
```suggestion
- 具体不同版本的兼容性说明，请查看各个版本的 [Release Note](/release-notes.md)。请根据各个版本的 Release Note 的兼容性更改调整集群的配置。
```

**Comment by lucklove** [comment]
LGTM

File: upgrade-tidb-using-tiup.md [hunk 7]
```diff
@@ -5,20 +5,24 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
...
 # 使用 TiUP 升级 TiDB
 
-本文档适用于使用 TiUP 从 TiDB 4.0 版本升级至 TiDB 5.0 版本，以及从 5.0 版本升级至后续修订版本。
+本文档适用于以下升级路径：
+- 使用 TiUP 从 TiDB 4.0 版本升级至 TiDB 5.1 及后续修订版本。
+- 使用 TiUP 从 TiDB 5.0 版本升级至 TiDB 5.1 及后续修订版本。
 
-如果原集群是 3.0 或 3.1 或更老的版本，需要先升级到 4.0 后再升级到 5.0，不可跨主版本升级。
+>  **注意：**
+>  如果原集群是 3.0 或 3.1 或更老的版本，不支持直接升级到 5.1 及后续修订版本。你需要先从老版本升级到 4.0 后，再从 4.0 升级到 5.1 及后续修订版本。
```
Comment (TomShawn) [inline upgrade-tidb-using-tiup.md:13]:
```suggestion
>  **注意：**
>
>  如果原集群是 3.0 或 3.1 或更老的版本，不支持直接升级到 5.1 及后续修订版本。你需要先从老版本升级到 4.0 后，再从 4.0 升级到 5.1 及后续修订版本。
```

File: upgrade-tidb-using-tiup.md [hunk 8]
```diff
@@ -5,20 +5,24 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
...
 
 ## 1. 升级兼容性说明
 
-- 不支持在升级后回退至 4.0 或更旧版本。
-- 使用 TiDB Ansible 管理的 4.0 版本集群，需要先按照 [4.0 版本文档的说明](https://docs.pingcap.com/zh/tidb/v4.0/upgrade-tidb-using-tiup)将集群导入到 TiUP (`tiup cluster`) 管理后，再按本文档说明升级到 5.0 版本。
-- 若要将 3.0 之前的版本升级至 5.0 版本：
+- TiDB 目前暂不支持版本降级或升级后回退，此类功能将在未来支持。
```
Comment (TomShawn) [inline upgrade-tidb-using-tiup.md:17]: 不建议对未来版本的支持作承诺。

Comment (lilyjazz) [reply upgrade-tidb-using-tiup.md:17]: 好的～

Comment (lilyjazz) [inline upgrade-tidb-using-tiup.md:17]:
```suggestion
- TiDB 目前暂不支持版本降级或升级后回退。
```

File: upgrade-tidb-using-tiup.md [hunk 9]
```diff
@@ -5,20 +5,25 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
...
 
-本文档适用于使用 TiUP 从 TiDB 4.0 版本升级至 TiDB 5.0 版本，以及从 5.0 版本升级至后续修订版本。
+本文档适用于以下升级路径：
+- 使用 TiUP 从 TiDB 4.0 版本升级至 TiDB 5.1 及后续修订版本。
+- 使用 TiUP 从 TiDB 5.0 版本升级至 TiDB 5.1 及后续修订版本。
 
-如果原集群是 3.0 或 3.1 或更老的版本，需要先升级到 4.0 后再升级到 5.0，不可跨主版本升级。
+>  **注意：**
+>
+>  如果原集群是 3.0 或 3.1 或更老的版本，不支持直接升级到 5.1 及后续修订版本。你需要先从老版本升级到 4.0 后，再从 4.0 升级到 5.1 及后续修订版本。
```
Comment (TomShawn) [inline upgrade-tidb-using-tiup.md:14]:
```suggestion
>  如果原集群是 3.0 或 3.1 或更早的版本，不支持直接升级到 5.1 及后续修订版本。你需要先从早期版本升级到 4.0 后，再从 4.0 升级到 5.1 及后续修订版本。
```

**Review by TomShawn — APPROVED** [review]
Rest LGTM

File: upgrade-tidb-using-tiup.md [hunk 10]
```diff
@@ -5,20 +5,25 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
 
 # 使用 TiUP 升级 TiDB
 
-本文档适用于使用 TiUP 从 TiDB 4.0 版本升级至 TiDB 5.0 版本，以及从 5.0 版本升级至后续修订版本。
+本文档适用于以下升级路径：
```
Comment (TomShawn) [inline upgrade-tidb-using-tiup.md:8]:
```suggestion
本文档适用于以下升级路径：

```

File: upgrade-tidb-using-tiup.md [hunk 11]
```diff
@@ -5,20 +5,25 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
...
+- 使用 TiUP 从 TiDB 4.0 版本升级至 TiDB 5.1 及后续修订版本。
+- 使用 TiUP 从 TiDB 5.0 版本升级至 TiDB 5.1 及后续修订版本。
 
-如果原集群是 3.0 或 3.1 或更老的版本，需要先升级到 4.0 后再升级到 5.0，不可跨主版本升级。
+>  **注意：**
```
Comment (TomShawn) [inline upgrade-tidb-using-tiup.md:12]:
```suggestion
> **注意：**
```

File: upgrade-tidb-using-tiup.md [hunk 12]
```diff
@@ -5,20 +5,26 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
...
+- TiDB 目前暂不支持版本降级或升级后回退。
+- 使用 TiDB Ansible 管理的 4.0 版本集群，需要先按照 [4.0 版本文档的说明](https://docs.pingcap.com/zh/tidb/v4.0/upgrade-tidb-using-tiup)将集群导入到 TiUP (`tiup cluster`) 管理后，再按本文档说明升级到 5.1 版本及后续修订版本。
+- 若要将 3.0 之前的版本升级至 5.1 版本：
     1. 首先[通过 TiDB Ansible 升级到 3.0 版本](https://docs.pingcap.com/zh/tidb/v3.0/upgrade-tidb-using-ansible)。
     2. 然后按照 [4.0 版本文档的说明](https://docs.pingcap.com/zh/tidb/v4.0/upgrade-tidb-using-tiup)，使用 TiUP (`tiup cluster`) 将 TiDB Ansible 配置导入。
     3. 将集群升级至 4.0 版本。
-    4. 按本文档说明将集群升级到 5.0 版本。
+    4. 按本文档说明将集群升级到 5.1 版本。
 - 支持 TiDB Binlog，TiCDC，TiFlash 等组件版本的升级。
+- 具体不同版本的兼容性说明，请查看各个版本的 [Release Note](/release-notes.md)。请根据各个版本的 Release Note 的兼容性更改调整集群的配置。
```
Comment (TomShawn) [inline upgrade-tidb-using-tiup.md:27]:
```suggestion
- 具体不同版本的兼容性说明，请查看各个版本的 [Release Note](/releases/release-notes.md)。请根据各个版本的 Release Note 的兼容性更改调整集群的配置。
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**